crates/bevy_render/src/lib.rs

```diff
@@ -57,6 +61,8 @@ pub mod stage {
     pub static RENDER_RESOURCE: &str = "render_resource";
     /// Stage where Render Graph systems are run. In general you shouldn't add systems to this stage manually.
     pub static RENDER_GRAPH_SYSTEMS: &str = "render_graph_systems";
+    /// Compute stage where compute systems are executed.
+    pub static COMPUTE: &str = "compute";
```

Comment:
Ideally I think we can throw all of the compute work onto it's own queue. We don't specifically need a stage, but we need to make sure that compute runs before any draw calls.

Reply:
Does compute need to be "queued up in a resource/component" before draw calls are "queued up in a resource/component"?

I imagine order would matter when it comes to creating command buffers / submitting draw calls, but thats all deferred until the render graph execution.

Reply:
> Does compute need to be "queued up in a resource/component" before draw calls are "queued up in a resource/component"?

Yes because you can have multiple dispatches.

> I imagine order would matter when it comes to creating command buffers / submitting command buffers, but thats all deferred until the render graph execution.

I think I need to look at the render graph a bit closer. I was assuming a graph node is a new command buffer?